samples/Maui/CounterApp/CounterApp.fsproj

```diff
@@ -102,4 +97,6 @@
 		<PackageReference Include="FSharp.Core" Version="6.0.5" />
 	</ItemGroup>
 
+	<Target Name="XamlPreCompile"></Target>

```

Comment:
If you don't have a C# project the WinUI targets still import their targets as if the C# xaml provider was imported. This target includes an expectation that `XamlPreCompile` exists so this just provides the target so things can execute.